### PR TITLE
add deployment worker size limits for GCP

### PR DESCRIPTION
**Pull request by afastronomer**
changes were done regarding #692

### CODE REVIEW

File: astro/resource-reference-gcp.md [hunk 1]
```diff
@@ -51,3 +51,22 @@ Astro supports different GCP machine types. Machine types comprise of varying co
 - e2-standard-8
 
 For detailed information on each instance type, see [GCP documentation](https://cloud.google.com/compute/docs/machine-types). If you're interested in a machine type that is not on this list, reach out to [Astronomer Support](https://support.astronomer.io/). Not all machine types are supported in all GCP regions.
+
+## Deployment Worker Size Limits
+
+In addition to setting a node instance type for each Cluster, you can configure a unique worker size for each Deployment within a Cluster. Worker size can be specified at any time in the **Worker Resources** field in the Deployment view of the Cloud UI. You can select any worker size up to 65 AU (6.5 CPUs, 24.4 GiB memory) as long as the worker size is supported by the node instance type selected for the Cluster. If you attempt to provision a worker size that isn't supported by your Cluster's instance type, you will see an error in the Cloud UI.
+
+This table lists the maximum worker size that is supported on Astro for each node instance type. These numbers may vary slightly over time.
```
Comment (kwcanuck) [inline astro/resource-reference-gcp.md:59]: Remove  _These numbers may vary slightly over time_

File: astro/resource-reference-aws.md [hunk 2]
```diff
@@ -214,7 +214,7 @@ This table lists the maximum worker size that is supported on Astro for each nod
...
 | t3.2xlarge         | 67         | 6.7 CPUs  | 25.1 GiB MEM |
 
-The maximum supported worker size on Astro is currently 400 AU, which means that an Astro Cluster may not make full use of the CPU and memory capacity of some node instance types on this list. If your team is interested in using an instance type that supports a worker size limit higher than 400 AU, reach out to [Astronomer Support](https://support.astronomer.io). For more information on configuring worker size on Astro, see [Configure a Deployment](configure-deployment.md#worker-resources).
+The maximum supported worker size on Astro is currently 400 AU, which means that an Astro Cluster may not make full use of the CPU and memory capacity of some node instance types on this list. If your team is interested in using an instance type that supports a worker size limit higher than 400 AU, reach out to [Astronomer Support](https://support.astronomer.io). For more information on configuring worker size on Astro, see [Configure a Deployment](configure-deployment.md#worker-resources). The maximum worker size values may increase or decrease as the CPU and memory requirements of Astro cloud change.
```
Comment (paolaperaza) [inline astro/resource-reference-aws.md:217]:
```suggestion
The maximum supported worker size on Astro is currently 400 AU, which means that an Astro Cluster may not make full use of the CPU and memory capacity of some node instance types on this list. If your team is interested in using an instance type that supports a worker size limit higher than 400 AU, reach out to [Astronomer Support](https://support.astronomer.io). For more information on configuring worker size on Astro, see [Configure a Deployment](configure-deployment.md#worker-resources).
```

Hm adding the statement here feels like an after thought - let's have it up top where we introduce these values?

File: astro/resource-reference-aws.md [hunk 3]
```diff
@@ -164,9 +164,9 @@ A single Cluster on Astro cannot currently be configured with more than one node
...
+In addition to setting a node instance type for each Cluster, you can configure a unique worker size for each Deployment within a Cluster. Worker size can be specified at any time in the **Worker Resources** field in the Deployment view of the Cloud UI. You can select any worker size up to 400 AU (40 CPUs, 150 GiB memory) as long as the worker size is supported by the node instance type selected for the Cluster. When you attempt to provision a worker size that isn't supported by the Cluster instance type, an error message appears in the Cloud UI.
 
-This table lists the maximum worker size that is supported on Astro for each node instance type. These numbers may vary slightly over time.
+This table lists the maximum worker size that is supported on Astro for each node instance type.
```
Comment (paolaperaza) [inline astro/resource-reference-aws.md:169]:
```suggestion
This table lists the maximum worker size that is supported on Astro for each node instance type. The maximum worker size values may increase or decrease as the CPU and memory requirements of the Astro application change.
```

Note: We actually don't say `Astro cloud` (I know, it's confusing). Let's just keep it as either `Astro` or `Astro application`

File: astro/resource-reference-gcp.md [hunk 4]
```diff
@@ -51,3 +51,24 @@ Astro supports different GCP machine types. Machine types comprise of varying co
...
+In addition to setting a node instance type for each Cluster, you can configure a unique worker size for each Deployment within a Cluster. Worker size can be specified at any time in the **Worker Resources** field in the Deployment view of the Cloud UI. You can select any worker size up to 65 AU (6.5 CPUs, 24.4 GiB memory) as long as the worker size is supported by the node instance type selected for the Cluster. When you attempt to provision a worker size that isn't supported by the Cluster instance type, an error message appears in the Cloud UI.
+
+This table lists the maximum worker size that is supported on Astro for each node instance type.
+
+| Node Instance Type | Maximum AU | CPU       | Memory       |
+|--------------------|------------|-----------|--------------|
+| e2-standard-4      | 25         | 2.5 CPUs  | 9.4  GiB MEM |
+| e2-standard-8      | 65         | 6.5 CPUs  | 24.4 GiB MEM |
+
+The maximum worker size values may increase or decrease as the CPU and memory requirements of Astro cloud change.
```
Comment (paolaperaza) [inline astro/resource-reference-gcp.md:66]:
```suggestion
If your team is interested in using an instance type that supports a worker size limit higher than 65 AU, reach out to [Astronomer Support](https://support.astronomer.io). For more information on configuring worker size on Astro, see [Configure a Deployment](configure-deployment.md#worker-resources).
```

Let's link out to support + the other relevant doc here (as it's also linked in AWS), and then pull up the statement on max worker size up top where we introduce these values (I think it's more applicable there)

**Review by paolaperaza — COMMENTED** [review]
Thanks @kwcanuck! Sorry, I took another look and have a couple (actually) final minor comments. LMK what you think.

File: astro/resource-reference-gcp.md [hunk 5]
```diff
@@ -51,3 +51,24 @@ Astro supports different GCP machine types. Machine types comprise of varying co
 - e2-standard-8
 
 For detailed information on each instance type, see [GCP documentation](https://cloud.google.com/compute/docs/machine-types). If you're interested in a machine type that is not on this list, reach out to [Astronomer Support](https://support.astronomer.io/). Not all machine types are supported in all GCP regions.
+
+## Deployment Worker Size Limits
+
+In addition to setting a node instance type for each Cluster, you can configure a unique worker size for each Deployment within a Cluster. Worker size can be specified at any time in the **Worker Resources** field in the Deployment view of the Cloud UI. You can select any worker size up to 65 AU (6.5 CPUs, 24.4 GiB memory) as long as the worker size is supported by the node instance type selected for the Cluster. When you attempt to provision a worker size that isn't supported by the Cluster instance type, an error message appears in the Cloud UI.
+
+This table lists the maximum worker size that is supported on Astro for each node instance type.
```
Comment (paolaperaza) [inline astro/resource-reference-gcp.md:59]:
```suggestion
This table lists the approximate maximum worker size that is supported on Astro for each node instance type. Maximum worker size values may increase or decrease over time as the system requirements of Astro change.
```

File: astro/resource-reference-gcp.md [hunk 6]
```diff
@@ -51,3 +51,24 @@ Astro supports different GCP machine types. Machine types comprise of varying co
 - e2-standard-8
 
 For detailed information on each instance type, see [GCP documentation](https://cloud.google.com/compute/docs/machine-types). If you're interested in a machine type that is not on this list, reach out to [Astronomer Support](https://support.astronomer.io/). Not all machine types are supported in all GCP regions.
+
+## Deployment Worker Size Limits
+
+In addition to setting a node instance type for each Cluster, you can configure a unique worker size for each Deployment within a Cluster. Worker size can be specified at any time in the **Worker Resources** field in the Deployment view of the Cloud UI. You can select any worker size up to 65 AU (6.5 CPUs, 24.4 GiB memory) as long as the worker size is supported by the node instance type selected for the Cluster. When you attempt to provision a worker size that isn't supported by the Cluster instance type, an error message appears in the Cloud UI.
```
Comment (paolaperaza) [inline astro/resource-reference-gcp.md:57]:
```suggestion
In addition to setting a node instance type for each Cluster, you can configure a unique worker size for each Deployment within a Cluster. Worker size can be specified at any time in the **Worker Resources** field in the Deployment view of the Cloud UI. For Deployments on GCP, you can select any worker size up to 65 AU (6.5 CPUs, 24.4 GiB memory) as long as the worker size is supported by the node instance type selected for the Cluster. When you attempt to provision a worker size that isn't supported by the Cluster instance type, an error message appears in the Cloud UI.
```

File: astro/resource-reference-aws.md [hunk 7]
```diff
@@ -164,9 +164,9 @@ A single Cluster on Astro cannot currently be configured with more than one node
...
+In addition to setting a node instance type for each Cluster, you can configure a unique worker size for each Deployment within a Cluster. Worker size can be specified at any time in the **Worker Resources** field in the Deployment view of the Cloud UI. You can select any worker size up to 400 AU (40 CPUs, 150 GiB memory) as long as the worker size is supported by the node instance type selected for the Cluster. When you attempt to provision a worker size that isn't supported by the Cluster instance type, an error message appears in the Cloud UI.
 
-This table lists the maximum worker size that is supported on Astro for each node instance type. These numbers may vary slightly over time.
+This table lists the maximum worker size that is supported on Astro for each node instance type. The maximum worker size values may increase or decrease as the CPU and memory requirements of the Astro application change.
```
Comment (paolaperaza) [inline astro/resource-reference-aws.md:169]:
```suggestion
This table lists the maximum worker size that is supported on Astro for each node instance type. Maximum worker size values may increase or decrease over time as the system requirements of Astro change.
```

**Pull request merged**